src/lib/external/balancer/router/interfaces.ts

```diff
@@ -13,9 +15,9 @@ export enum SwapKind {
 
 export interface BatchSwapStep {
     poolId: string;
-    assetInIndex: number;
-    assetOutIndex: number;
-    amount: string;
+    assetInIndex: BigInt;
```

Comment:
should be `bigint` not `BigInt`, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#type_information

Reply:
Multiple spots where you use `BigINt` in the PR. Won't mark all of them.